internal/resource/informer.go

```diff
@@ -185,7 +209,33 @@ func (ri *resourceInformer) Refresh() error {
 		}
 
 	}
-	// Node
+
+	containersNoPod := []string{}
+	if ri.podInformer != nil {
+		for _, container := range containersRunning {
```

Comment:
missing test? In a follow up PR,  could you please add test to test this functionality?